tools/project_manager/diff.php

```diff
@@ -61,6 +62,19 @@
     $R_label = $R_round->id;
 }
 
+$L_format = is_format_round($L_label);
+$R_format = is_format_round($R_label);
+$format = array_get($_GET, "format", "default");
```

Comment:
Move this to the top of the file with the other $_GET validation. It should also be using `get_enumerated_param()` with a default parameter instead of a blanket array_get().